CMakeLists.txt

```diff
@@ -18,7 +18,8 @@ find_qt(COMPONENTS Core Widgets Svg Network)
 find_package(nlohmann_json 3 REQUIRED)
 
 # Find qrcodegencpp
-find_package(Libqrcodegencpp REQUIRED)
+set(CMAKE_FIND_PACKAGE_PREFER_CONFIG ON)
```

Comment:
Should this be toggled back OFF after the subsequent `find_package` call, or does it not matter?

Reply:
It doesn't matter _effectively_ but logically it should be reset, because I wouldn't say that we can make that same conscious decision for the other dependencies.